templates/tweet/follow_list.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'twitter/base.html' %}
+{% load static %}
+
+{% block title %}
+followlist
+{% endblock title %}
+
+{% block contents %}
+<div class="ui text container"><br>
+  {% for user in object_list %}
+    <div class="alert alert-info" role="alert">
+      <table class="ui celled padded table">
+        <tr>
+          <td>nickname</td>
+          <td>{% if user.nickname %}{{ user.nickname }}{% else %}I don't have a name yet. ({{user.pk}}){% endif %}</td>
```

Comment:
```suggestion
          <td>{% if user.nickname %}{{ user.nickname }}{% else %}I don't have a name yet. ({{ user.pk }}){% endif %}</td>
```
他もちょくちょく空白が抜けてるので，チェックしてみてください！

Reply:
同様のミスをいくつもしていましたが、確認できたところは直しました。